eox_nelp/stats/metrics.py

```diff
@@ -134,3 +140,49 @@ def get_courses_metrics(tenant):
     metrics = [get_course_metrics(course.id) for course in courses]
 
     return {"total_courses": courses.count(), "metrics": metrics}
+
+
+def get_course_certificates_metric(course_key):
+    """
+    Returns the total of certificates in a course.
+
+    Args:
+        course_key<opaque-key>: Course identifier.
+
+    Return:
+        <Dictionary>: Contains certificates of course metric.
+        {
+                "verified": {...},
+                "honor": {...},
+                "audit": {},
+                "professional": {...},
+                "no-id-professional": {
+                    "downloadable": 5,
+                    "notpassing": 4,
+                },
+                "masters": {...},
+                "executive-education": {...}
+                "paid-executive-education":{...},
+                "paid-bootcamp": {...},
+                "total": 0
```

Comment:
since you are splitting the certificates based on their status the total metric should reflect also that division

Reply:
Could you explain this one a little more, I mean I am setting the total without the division. 
So the division would reflect something like the total: 
``` python 
"total" : {
"no-id-professional": 5,
"audit": 5
}
```
Is that what you say?
Because if is that I prefer to set the total of each one inside each key?? But I didn't find it necessary due you only have to sum the field of each one

Reply:
I didn't say mode 
```
                "total": {
                    "downloadable": 5,
                    "notpassing": 4,
                },
```

Reply:
> I didn't say mode
> 
> ```
>                 "total": {
>                     "downloadable": 5,
>                     "notpassing": 4,
>                 },
> ```

@andrey-canon  what about this approach ?
![Peek 2023-09-12 17-55](https://github.com/eduNEXT/eox-nelp/assets/51926076/3a6ea5ed-9e0d-45b0-ba5b-1511e3d7e2fc)

Reply:
did you remove the modes ?